### PR TITLE
[RU] - some fix

### PR DESCRIPTION
**Pull request by Sinitsaa**
A few fixes. By next wednesday there will be more.
Merge, please.

### CODE REVIEW

File: translated files/Dialogue/Mid-Mission/incident_common.csv [hunk 1]
```diff
@@ -1,209 +1,209 @@
...
-QuestFailure#5,"""仕方ないですね……帰還しましょう。"""
+QuestSuccess#0,"""/face2 Пожалуйста, немедленно<br>возвращайтесь."""
+QuestFailure#0,"""/face4 Ничего не поделать.<br>Возвращайтесь."""
+QuestSuccess#1,"""Предоставленно разрешение на<br>возвращение.<br>Вы можете вернуться."""
+QuestFailure#1,"""Покидайте свою миссию<br>и возвращайтесь.<br>Вы сделали всё возможное."""
+QuestSuccess#2,"""/face2 Хорошая работа!<br>Пожалуйста, возвращайтесь!"""
+QuestFailure#2,"""/face6 Что ж, в любом случае,<br>пожалуйста, возвращайтесь!"""
+QuestSuccess#3,"""Миссия завершена.<br>Немедленно возвращайтесь."""
+QuestFailure#3,"""Подтверждён провал миссии.<br>Немедленно возвращайтесь."""
+QuestSuccess#4,"""Хорошая работа! <br>Пожалуйста, возвращайтесь!""""
```
Comment (DabirSA) [inline translated files/Dialogue/Mid-Mission/incident_common.csv]: Extra " here.

File: translated files/Dialogue/Mid-Mission/incident_common.csv [hunk 2]
```diff
@@ -1,209 +1,209 @@
...
+QuestFailure#4,"""У нас не было выбора...<br>возвращайтесь."""
+QuestSuccess#5,"""Хорошая работа! <br>Пожалуйста, возвращайтесь!""""
```
Comment (DabirSA) [inline translated files/Dialogue/Mid-Mission/incident_common.csv]: Extra " here.

File: translated files/Dialogue/Mid-Mission/incident_common.csv (hunk 1 above)
Comment (Sinitsaa) [reply translated files/Dialogue/Mid-Mission/incident_common.csv]: Thx :3

File: translated files/UI/common.csv [hunk 3]
```diff
@@ -1377,8 +1377,8 @@ Change_Text#0,"""Введите пароль (<yellow>5 цифр<c>)"""
 Change_Menu1#0,"""Старый пароль:"""
 Change_Menu2#0,"""Новый пароль:"""
 Change_Menu3#0,"""Повторите пароль:"""
-Change_Notice#0,"""安全のため、右に表示されているボタンから<br>マウスでパスワードを入力してください。<br>パスワードをリセットする場合は<br>「パスワードのリセット」を選択してください。"""
-Change_NoticeV#0,"""登録したアイテム取引パスワードは<br>アイテムを取引する際に必要となります。<br><br>パスワードをリセットする場合は<br>「パスワードのリセット」を選択してください。"""
+Change_Notice#0,"""Для сохранности вашего аккаунта,<br>пожалуйста, введите пароль передачи<br>предметов.<br>Для сброса пароля, пожалуйста,<br>выберите \""Сбросить пароль\"".""
```
Comment (DabirSA) [inline translated files/UI/common.csv]: Missing " here.

Comment (Sinitsaa) [reply translated files/UI/common.csv]: Thx :3

File: translated files/Dialogue/Mid-Mission/incident_common.csv (hunk 2 above)
Comment (Sinitsaa) [reply translated files/Dialogue/Mid-Mission/incident_common.csv]: Thx :3

**Comment by alama** [comment]
Can I ask WHY unmodified Files from the Files folder are moved out?

**Comment by Sinitsaa** [comment]
They will be modified later. I want to merge before technical work.

**Comment by alama** [comment]
You may want to check:
[translated files/Quests/Evil Dragon/quest_00050770.csv:Explanation#0](https://github.com/Arks-Layer/PSO2ENPatchCSV/commit/6a227782e0f6de7b9d2f9ed8343a3d9d6b8e42cb#diff-b69a320aeb92d06a4f5296695e67aff9R5)
[translated files/Quests/Evil Dragon/quest_00050770.csv:TargetExplanation#0](https://github.com/Arks-Layer/PSO2ENPatchCSV/commit/6a227782e0f6de7b9d2f9ed8343a3d9d6b8e42cb#diff-b69a320aeb92d06a4f5296695e67aff9R14)
[translated files/Quests/Evil Dragon/quest_00170320.csv:Explanation#0](https://github.com/Arks-Layer/PSO2ENPatchCSV/commit/6a227782e0f6de7b9d2f9ed8343a3d9d6b8e42cb#diff-9da79486332f48406d6f05c90287baddR5)
[translated files/Quests/Evil Dragon/quest_00170320.csv:TargetExplanation#0](https://github.com/Arks-Layer/PSO2ENPatchCSV/commit/6a227782e0f6de7b9d2f9ed8343a3d9d6b8e42cb#diff-9da79486332f48406d6f05c90287baddR14)

They had been changed on the JP side

**Pull request merged**